src/blockchains/construct_worker.ts

```diff
@@ -0,0 +1,34 @@
+'use strict';
+import { BaseWorker } from '../lib/worker_base';
+import { CardanoWorker } from './cardano/cardano_worker';
+import { ERC20Worker } from './erc20/erc20_worker';
+import { ETHWorker } from './eth/eth_worker';
+import { ETHBlocksWorker } from './eth_blocks/eth_blocks_worker';
+import { MaticWorker } from './matic/matic_worker';
+import { ReceiptsWorker } from './receipts/receipts_worker';
+import { UTXOWorker } from './utxo/utxo_worker';
+import { XRPWorker } from './xrp/xrp_worker';
+
+export function constructWorker(blockchain: string, settings: any): BaseWorker {
```

Comment:
We used to initialize the class of the worker dynamically like this:

`const worker = require(`./blockchains/${BLOCKCHAIN}/${BLOCKCHAIN}_worker`);`

move away from this to get better type checking.